packages/rxjs/.eslintrc.json

```diff
@@ -3,10 +3,15 @@
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended"
   ],
+  "parserOptions": {
+    "project": ["./tsconfig.json"]
+  },
   "rules": {
     "@typescript-eslint/no-explicit-any": "off",
     "@typescript-eslint/no-unused-vars": "off",
     "@typescript-eslint/no-unsafe-declaration-merging": "off",
+    "@typescript-eslint/consistent-type-imports": "warn",
+    "@typescript-eslint/consistent-type-exports": "warn",
```

Comment:
Question: Should these be "error" instead of warn?

Reply:
#7391. I also discovered that eslint also checks dist. I will create a separate PR.